BowlingBall.Tests/GameFixture.cs

```diff
@@ -5,10 +5,73 @@ namespace BowlingBall.Tests
 {
     public class GameFixture
     {
+        private Game game;
+
+        protected void SetUp()
+        {
+            game=new Game();
+        }
+
+        private void RollMany(int n,int pins)
+        {
+            for(int i=0;i<n;i++)
+                game.Roll(pins);
+        }
+
+        private void RollSpare()
+        {
+            game.Roll(5);
+            game.Roll(5);
+        }
+        private void RollStrike()
+        {
+            game.Roll(10);
+        }
         [Fact]
-        public void DummyTest()
+        public void TestGutterGame()

```

Comment:
I don't see test case for below scenarios : In the tenth frame a player who rolls a spare or strike is allowed to roll the extra balls to complete the frame. However no more than three balls can be rolled in tenth frame.